### PR TITLE
edit README

### PR DESCRIPTION
**Pull request by Clap404**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Spelling and readability of README files. 
| Type?             | improvement
| BC breaks?        |  no
| Deprecations?     |  no
| Fixed ticket?     | none
| Sponsor company   | PrestaShop
| How to test?      | Check that information properly enters your brain when read.

### CODE REVIEW

File: examples/ngrok-tunnel/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Example: ngrok tunnel example
 
-Ngrok is a handy solution to expose your local environment to the Web. You can get information about this third-party solution on their official web page: https://ngrok.com.
+[Ngrok](https://ngrok.com) is a handy http tunnel you can use to expose your local environment to the Web and inspect incoming requests.
 
 ## Test the example
 
-1. First you will have to Sign up (it's free). Once it's done, on the left menu clic on "Getting Started > Your Authtoken".
```
Comment (Clap404) [inline examples/ngrok-tunnel/README.md:7]: Ngrok is not free for everyone / every use case.

File: examples/develop-prestashop/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Example: develop prestashop
 
-Flashlight can help you bootstraping an environment to perform live development of PrestaShop itself. We hope the tool to help your contribution, with a fast and switchable environment!
+Flashlight can help you bootstrap an environment to perform live development of PrestaShop itself. We hope the tool to help your contribution, with a fast and flexible environment!
```
Comment (Clap404) [inline examples/develop-prestashop/README.md:3]: I assume "switchable" means we can easily swap prestashop / php versions ?

File: examples/ngrok-tunnel/README.md (hunk 1 above)
Comment (jokesterfr) [reply examples/ngrok-tunnel/README.md:7]: It is free for most common use cases. I'm trying to convince the enduser to try it

Comment (jokesterfr) [reply examples/ngrok-tunnel/README.md:7]: I added this sentence: `For this simple use case the free plan is sufficient.`

**Comment by jokesterfr** [comment]
Thank you!

**Pull request merged**